faas/lambda_retrigger_model_output_add.py

```diff
@@ -43,7 +43,7 @@ def main():
 
                 s3_resource = boto3.resource("s3")
                 s3_object = s3_resource.Object(s3_bucket, key)
-                s3_object.metadata.update({"x-amz-meta-manual-update": update_date})
+                s3_object.metadata.update({"manual-update": update_date})
```

Comment:
@matthewcornell looks like Amazon automatically adds the `x-amz-meta` piece of the user-generated metadata tag. When I manually deleted the metadata tags on the `example-complex-forecast-hub` file below and re-ran the script, the tag appeared as expected:
![image](https://github.com/user-attachments/assets/e6665507-c0a6-4c51-836d-e150ba1de753)

And when I re-ran the script, it updated the existing tag with a new timestamp:

![image](https://github.com/user-attachments/assets/13083229-cbfd-4d06-b623-e06387a84805)